src/inmanta/app.py

```diff
@@ -465,6 +465,19 @@ def export_parser_config(parser: argparse.ArgumentParser, parent_parsers: abc.Se
         action="store_true",
         default=False,
     )
+    parser.add_argument(
+        "--bypass-base-version-check",
```

Comment:
I think we should improve the name. Throughout the code, I don't mind too much (though consistency is probably best), but here we really need to come at it from the end user perspective. "base version check" means little to them. The error they get doesn't mention it either.

So, think about the end user, what is it they're really trying to do, and what triggers them to do it?

Reply:
> The error they get doesn't mention it either.

I withdraw this part, I see you updated it. That makes it better, but I still think it can be improved.